rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -186,6 +177,29 @@ public static ISchedulerClient createInstance() {
                                                          new SessionHandler(client));
     }
 
+    private static String createJobSortParamsString(List<SortParameter<JobSortParameter>> jobSortParameters) {
+
+        String jobSortParametersAsString = "";
+
+        for (SortParameter<JobSortParameter> jobSortParameter : jobSortParameters) {
+            final JobSortParameter jobParameter = jobSortParameter.getParameter();
+            jobSortParametersAsString += jobParameter.name();
+
+            if (jobSortParameter.getSortOrder().isAscending()) {
+                jobSortParametersAsString += ASC_SUFFIX;
+            } else {
+                jobSortParametersAsString += DESC_SUFFIX;
+            }
+
+            jobSortParametersAsString += ",";
+        }
+
+        return Optional.ofNullable(jobSortParametersAsString)
```

Comment:
You can make `Optional.of()` instead of  `Optional.ofNullable` as you initialize with an empty String

Reply:
No more used in the next commit.